pp3/module/Application/src/Application/Controller/VerificationController.php

```diff
@@ -196,7 +196,7 @@ private function _sendNoGoNotification($verification, $comment) {
 
 If your plugin was verified successfully, it should be already available on the Plugin Portal Update Center by now:
 
-http://netbeans-vm.apache.org/pluginportal/data/'.$nbVersion.'/catalog.xml.gz
+https://plugins.netbeans.apache.org/data/'.$nbVersion.'/catalog.xml.gz
```

Comment:
This is the nogo message. So my understanding is, that in case of a no-go, there will be no publication. I would then remove lines 197, 198 and 199

Reply:
Good idea. Resolved by #33 /bbb8c7a9d1a6675826fcb705c1e915ef11078cc3 Thanks Matthias.